apps/src/templates/progress/progressHelpers.js

```diff
@@ -119,57 +131,106 @@ export function stageIsAllAssessment(levels) {
 
 /**
  * Summarizes stage progress data.
- * @param {[]} levelsWithStatus An array of objects each representing
- * students progress in a level
+ * @param {[]} studentProgress An object keyed by level id containing objects
+ * representing the student's progress in that level
+ * @param {[]} stageLevels An array of the levels in a stage
  * @returns {object} An object with a total count of levels in each of the
  * following buckets: total, completed, imperfect, incomplete, attempted.
  */
-export function summarizeProgressInStage(levelsWithStatus) {
+export function summarizeProgressInStage(studentProgress, stageLevels) {
```

Comment:
Mostly curious -- is "stageLevels" the same thing that's passed in the "levels" param to stageLocked() and other functions in this file?

Reply:
yes. i have updated the naming to be consistent throughout the file.